src/Util/FilterUtil.ts

```diff
@@ -117,7 +117,7 @@ class FilterUtil {
     const prop: any = _get(feature, 'properties[' + filter[1] + ']');
```

Comment:
Even if not related to this PR. This should be renamed to `featureValue`. Additonaly
```js
const filterValue = filter[2];
```
would add some clarification...

Reply:
Done in f9b4acb